src/main/java/com/ft/methodearticlemapper/model/EomFile.java

```diff
@@ -93,10 +93,10 @@ public URI getWebUrl() {
     }
     
     @JsonAnySetter
-    public void setAdditionalProperty(String name, String value) {
+    public void setAdditionalProperty(String name, Object value) {
         String mappedName = ADDITIONAL_PROPERTY_MAPPINGS.get(name);
-        if (mappedName != null) {
-            additionalProperties.put(mappedName, value);
+        if ((mappedName != null) && (value instanceof String)) {
+            additionalProperties.put(mappedName, (String)value);
```

Comment:
Just to confirm - do we only want additional properties which are String?

Reply:
For now, the only case where I expect this to be used is for `lastModified`, `publishReference` (UPP) or `draftReference` (PAC) - which are all strings.